### PR TITLE
ja: refresh ja.po file

### PR DESCRIPTION
**Pull request by kantasv**
Hi, ja-translators, #652 ! I've refreshed `ja.po` file as of today, Jan 8th 2024.
I used the following procedures. Could you review the diff? Thanks 😊 

```
MDBOOK_OUTPUT='{"xgettext": {"pot-file": "messages.pot"}}' \
  mdbook build -d po
```

```
msgmerge --update po/ja.po po/messages.pot
```

### CODE REVIEW

**Comment by kantasv** [comment]
FYI: POT-Creation-Date looks slightly different and not sure if it's WAI or not, that is:

**Current (81c0629)** in `ja.po`:
```
"POT-Creation-Date: 2024-01-08T14:58:00+09:00\n"
```
**Previous (0e8d3e7)** in `ja.po`:
```
"POT-Creation-Date: 2023-10-15T23:41:50Z\n"
```

Out of curiosity, checked other lang po.files:

```
bn.po:"POT-Creation-Date: 2023-09-06\n"
da.po:"POT-Creation-Date: 2023-11-30T15:58:18+01:00\n"
de.po:"POT-Creation-Date: 2023-09-19\n"
el.po:"POT-Creation-Date: 2023-08-26\n"
es.po:"POT-Creation-Date: 2023-09-11\n"
fa.po:"POT-Creation-Date: 2023-08-23\n"
fr.po:"POT-Creation-Date: 2023-12-08T18:20:47-05:00\n"
id.po:"POT-Creation-Date: 2023-08-26\n"
it.po:"POT-Creation-Date: 2023-10-24T09:24:14+01:00\n"
ja.po:"POT-Creation-Date: 2024-01-08T14:58:00+09:00\n"
ko.po:"POT-Creation-Date: 2023-09-24\n"
pl.po:"POT-Creation-Date: 2023-08-26\n"
pt-BR.po:"POT-Creation-Date: 2023-08-28\n"
ro.po:"POT-Creation-Date: 2023-11-07T13:34:38+02:00\n"
ru.po:"POT-Creation-Date: 2023-09-09\n"
tr.po:"POT-Creation-Date: 2023-10-24T20:37:45+03:00\n"
uk.po:"POT-Creation-Date: 2023-12-18T21:00:17-05:00\n"
zh-CN.po:"POT-Creation-Date: 2023-10-06T12:36:03-07:00\n"
zh-TW.po:"POT-Creation-Date: 2023-09-11 10:37-0700\n"
```

**Comment by kantasv** [comment]
Ok, TIL that Z at the end means zero-offset from UTC, RE: https://github.com/google/comprehensive-rust/pull/1676#issuecomment-1880443467

Btw, a test on GitHub Action has failed. I'll take a look and will get back. No review is required until I resolve the `Test / format` failure. Thanks!

https://github.com/google/comprehensive-rust/actions/runs/7443837831/job/20249295882?pr=1676

**Comment by kantasv** [comment]
**TL;DR** dprint on CI errored out for the refreshed `ja.po` but I have no idea how to fix it. Can I manually fix the entry so that it passes the CI, or is there any proper way to format it? Also, I found that dprint config on Actions CI was recently changed in https://github.com/google/comprehensive-rust/pull/1157 . @mgeisler , do you think it's related? 

The following has failed, and found the config file for this check, [dprint.json](https://github.com/google/comprehensive-rust/blob/main/dprint.json). 
```
      - name: Check formatting
        uses: dprint/check@v2.2
```

Seems `ja.po` file isn't formatted properly from the [dprint's log](https://github.com/google/comprehensive-rust/actions/runs/7443837831/job/20249295882?pr=1676). The error comes from the `ja.po` file:

```
from /home/runner/work/comprehensive-rust/comprehensive-rust/po/ja.po:
...
Found 1 not formatted file.
```

Excerpt from the dpring log (there are other msgids with similar format errors):
```
9179 9179| #:·src/modules/filesystem.md:18
9180 9180| msgid·""
     9181|-"Modules·defined·in·files·can·be·documented,·too,·using·\"inner·doc·comments"
     9182|-"\".·These·document·the·item·that·contains·them·--·in·this·case,·a·module."
9181     |+"Modules·defined·in·files·can·be·documented,·too,·using·\"inner·doc·"
9182     |+"comments\".·These·document·the·item·that·contains·them·--·in·this·case,·a·"
9183     |+"module."
9184 9183| msgstr·""
```

I also found a roll-out of recent changes regarding dprint in the GitHub Actions as in https://github.com/google/comprehensive-rust/pull/1157 . @mgeisler , do you think if it's related?

**Comment by mgeisler** [comment]
> **L;DR** dprint on CI errored out for the refreshed `ja.po` but I have no idea how to fix it.

Do you have `dprint` installed locally and did you run `dprint fmt`? That would be the way to fix it.

As you found out, we started letting `dprint` format all the Markdown files in #1157, but this should not matter for the `.po` file in question. The PO files have been required to be formatted a bit longer, perhaps two months or so. You should be able to format it following the instructions on https://github.com/google/comprehensive-rust/blob/main/CONTRIBUTING.md#formatting.

Please let me know if the instructions don't work! :smile: They have to be easy to follow, otherwise everybody will have a bad time going forward.

**Comment by kantasv** [comment]
Hi, @mgeisler thanks for clarification and thanks for adding instructions to `CONTRIBUTING.md` and `TRANSLATION.md`  as in #711 and #1372 . Sorry, I just didn't catch up those updates in documentation, but it was good to know! 

Now I have formatted `ja.po` file with `dprint`. Could someone review the diff when you have some time? Thanks!

#### Total steps for update ja.po so far
```
MDBOOK_OUTPUT='{"xgettext": {"pot-file": "messages.pot"}}' \
  mdbook build -d po
```

```
msgmerge --update po/ja.po po/messages.pot
```

```
dprint fmt po/ja.po
```

**Review by keiichiw — APPROVED** [review]
Sorry for the late reply. Thanks for the update!

**Comment by kantasv** [comment]
@keiichiw -san, thanks for the reveiw and the approval! Merging :)
